### ISSUE TITLE
WhatApp Cannt Connect With This

### ISSUE THREAD

**Issue by yhnu** [Skifary/c4msis#3]

**Comment by Skifary** [comment]
@buutuud no, i have tried adding whatsapp to the rule, but something work and something not, i'm not sure if `NEKit` has a way to intercept all requests. any advice here?

**Comment by buutuud** [comment]
@Skifary tun2sock may can solve it.we can try

**Comment by Skifary** [comment]
@buutuud okay, let me try

**Comment by LeeCenY** [comment]
WhatApp 即时通讯类是要走 UDP 转发

**Comment by LeeCenY** [comment]
WhatsApp instant messaging class is going to take UDP forwarding, I tried setting up UDP to connect, but it will drop in a few seconds

**Comment by yhnu** [comment]
@LeeCenY solve it now ?

**Comment by LeeCenY** [comment]
@buutuud Not yet. I just started playing, and NEKit didn't quite understand.

**Comment by v2rayU** [comment]
@LeeCenY  telegram Can't login
